src/features/planning/providers/read/fares-by-day.read.provider.ts

```diff
@@ -2,15 +2,15 @@ import { FactoryProvider } from '@angular/core';
 import { Observable } from 'rxjs';
 import { FareByDayPresentation } from '../../presentation';
 
-export const FARES_BY_DAY_READ: 'planning.fares-by-day.read' = 'planning.fares-by-day.read' as const;
+export const FARES_BY_DAY: 'planning.fares-by-day.read' = 'planning.fares-by-day.read' as const;
 
-export type FaresByDayRead = (date: Date) => Observable<FareByDayPresentation[]>;
+export type FaresByDay = (date: Date) => Observable<FareByDayPresentation[]>;
```

Comment:
```suggestion
export type FaresByDayQuery = (date: Date) => Observable<FareByDayPresentation[]>;
```
Tu vas me tuer...

Sur ma pierre tombale tu pourras écrire : 
À changé d'avis pour la 42ème fois car il trouvait que `FaresByDay` ressemblait plus à une forme array de `FareByDay` et que si on renomme `FareByDayPresentation` en `FareByDay` et qu'on le met ici ça porte trop à confusion
Repose en paix

Reply:
export type FaresByDay = (date: Date) => Observable<FareByDay[]>;

Reply:
Oui je tombais dessus à l'instant...

Reply:
Du coup Query plutot que Read ?

Reply:
Oui, ça ne te choque pas toi ?

Reply:
Quand je vois `FaresByDay` dans le code, je m'imagine que c'est un ; `export type FaresByDay = FareByDay[]`

Reply:
`export type FaresByDayQuery = (date: Date) => Observable<FareByDay[]>;`
Est plus clair que c'est un truc sur lequel il faut faire un call et il n'y a pas le `Read` que je ne trouve pas top